tests/unit/test_charm.py

```diff
@@ -388,22 +388,6 @@ def bundle_handler(args: ops.testing.ExecArgs) -> None:
     charm._on_anonymize_user_action(event)
 
 
-def test_handle_pebble_ready_event():
```

Comment:
This test is not doing what it says. The plan before the pebble ready was empty because of Redis integration.